samples/command/consumer/src/main.rs

```diff
@@ -80,59 +78,31 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     let addr: SocketAddr = consumer_authority.parse()?;
     let consumer_impl = consumer_impl::ConsumerImpl::default();
     let server_future =
-        Server::builder().add_service(ConsumerServer::new(consumer_impl)).serve(addr);
+        Server::builder().add_service(DigitalTwinConsumerServer::new(consumer_impl)).serve(addr);
     info!("The HTTP server is listening on address '{CONSUMER_ADDR}'");
 
-    // Obtain the DTDL for the send_notification command.
     info!("Sending a find_by_id request for entity id {} to the In-Vehicle Digital Twin Service URI {IN_VEHICLE_DIGITAL_TWIN_SERVICE_URI}",
         sdv::vehicle::cabin::infotainment::hmi::show_notification::ID);
     let mut client = DigitalTwinClient::connect(IN_VEHICLE_DIGITAL_TWIN_SERVICE_URI).await?;
     let request = tonic::Request::new(FindByIdRequest {
-        entity_id: String::from(sdv::vehicle::cabin::infotainment::hmi::show_notification::ID),
+        id: String::from(sdv::vehicle::cabin::infotainment::hmi::show_notification::ID),
     });
     let response = client.find_by_id(request).await?;
-    let dtdl = response.into_inner().dtdl;
+    let response_inner = response.into_inner();
     debug!("Received the response for the find_by_id request");
+    info!("response_payload: {:?}", response_inner.entity_access_info);
 
-    debug!("Parsing the DTDL.");
-    let mut parser = ModelParser::new();
-    let json_texts = vec![dtdl];
-    let model_dict_result = parser.parse(&json_texts);
-    if let Err(error) = model_dict_result {
-        panic!("Failed to parse the DTDL: {error}");
-    }
-    let model_dict = model_dict_result.unwrap();
-    debug!("The DTDL parser has successfully parsed the DTDL");
-
-    // Create the id (as a DTMI) for the show-notification command.
-    let show_notification_command_id: Option<Dtmi> =
-        create_dtmi(sdv::vehicle::cabin::infotainment::hmi::show_notification::ID);
-    if show_notification_command_id.is_none() {
-        panic!("Unable to create the dtmi");
-    }
-
-    // Get the entity from the DTDL for the show-notification command.
-    let entity_result = model_dict.get(&show_notification_command_id.unwrap());
-    if entity_result.is_none() {
-        panic!("Unable to find the entity");
-    }
-    let entity = entity_result.unwrap();
-
-    // Get the URI property from the entity.
-    let uri_property_result = entity.undefined_properties().get(sdv::property::uri::ID);
-    if uri_property_result.is_none() {
-        panic!("Unable to find the URI property");
+    let provider_uri;
```

Comment:
The match statement below can be replaced with: ``
let content = response_inner.entity_access_info.expect("Did not find an entity for the ShowNotification command");
let provider_uri = content.endpoint_info_list[0].uri.clone();
``